[Alerting][Docs] Adding link to ES docs for CCS setup (#141995)
elastic/kibana: docs/user/alerting/alerting-setup.asciidoc

function. Conversely, if a user with greater or administrator privileges 
modifies the rule, it will begin running with increased privileges.
==============================================

[float]
[[alerting-ccs-setup]]

> === CCS

Reviewer: ```suggestion
=== {ccs-cap}
```

> If you want to use alerting rules in a cross-cluster setup, you must configure 

Reviewer: ```suggestion
If you want to use alerting rules with {ccs}, you must configure 
```

> {ref}/remote-clusters-privileges.html#clusters-privileges-ccs-kibana[privileges for cross-cluster search and Kibana]

Reviewer: ```suggestion
{ref}/remote-clusters-privileges.html#clusters-privileges-ccs-kibana[privileges for {ccs-init} and {kib}].
```
